utils/yoke/src/erased.rs

```diff
@@ -20,6 +21,12 @@ use alloc::rc::Rc;
 pub trait ErasedDestructor: 'static {}
 impl<T: 'static> ErasedDestructor for T {}
 
+/// A type-erased Cart that has `Arc` semantics
+///
+/// See the docs of [`Yoke::erase_arc_cart()`](crate::Yoke::erase_rc_cart) for more info.
+///
+/// Available with the `"alloc"` feature enabled.
+pub type ErasedArcCart = Arc<dyn ErasedDestructor + Send + Sync>;
```

Comment:
Note: We'll need to release `yoke` if we merge this PR before 1.0 (CC @Manishearth)